eng/native/functions.cmake

```diff
@@ -89,6 +89,10 @@ function(find_unwind_libs UnwindLibs)
       find_library(UNWIND_ARCH NAMES unwind-x86_64)
     endif()
 
+    if(CLR_CMAKE_HOST_ARCH_S390X)
```

Comment:
a nit: this (the condition on callsite `if (CLR_CMAKE_USE_SYSTEM_LIBUNWIND)`) is unused on platforms that use the in-tree libunwind (like linux-s390x).

If you would revert it, please leave a `# comment` on top of this function to make it clearer for future reader.

Reply:
Ah, actually if you intentionally want to use system libunwind, then you can pass `./build.sh -cmakeargs CLR_CMAKE_USE_SYSTEM_LIBUNWIND=1` or `./build.sh --portable=false`. nvm, disregard please. :)

Reply:
Yes, exactly.  This is just to allow that use case.